scripts/linters/test_files/invalid_merge_conflict.py

```diff
@@ -25,4 +25,4 @@
 'this is some content to mess with content to append.'
 =======
 'totally different content to merge later.'
->>>>>>> new_branch_to_merge_later
+>>>>>>> new_branch_to_merge_later
```

Comment:
Just one question, any reason for removing the newline from EOF.

Reply:
Nope, was a mistake -- fixed.